geometry/path.cc

```diff
@@ -19,8 +18,6 @@ Path::~Path() = default;
 
 std::tuple<size_t, size_t> Path::Polyline::GetContourPointBounds(
     size_t contour_index) const {
-  FML_DCHECK(contour_index < contours.size());
```

Comment:
Filed https://github.com/flutter/flutter/issues/99727. This method isn't safe as written anyway.